### ISSUE TITLE
[Documentation] Create metrics-tracking sheet & sharing strategies 

### ISSUE THREAD

**Issue by kategarklavs** [18F/web-design-standards#1468]
## Description

This is a subtask of #1401. Our team needs an easy-to-find spot to record monthly content metrics — I'm thinking a Google sheet might be the way to go (and am also open to all suggestions).

For this task, we'll do the following:

- [x] Come up with a method for recording metrics
- [x] Identify how best to share metrics with the team

**Comment by jameshupp** [comment]
<3 Google sheets, so that's my vote, as well. I'm playing around with this today to see if I can't start down that road.

**Comment by kategarklavs** [comment]
@jameshupp Sweet! I'm going to review/address comments in our metrics-tracking doc, and then I can help out with this, too 👍

**Comment by jameshupp** [comment]
Yesterday @kategarklavs and I discussed our initial attempt at this, which seemed to involve too many things. Not enough time available to team members, and trying to put our arms around too much stuff to improve things quickly. So we narrowed our focus, and our current suggestion is:

- [Update the comms log](https://github.com/18F/web-design-standards/issues/1467) to capture more of our communications than one-to-one emails. This will let us synthesize concerns more clearly.
- Add questions to the interview guide about some of the specific communications channels that concern us so we have those inputs alongside the rest.

The types of issues we are most interested in are those related to comprehensibility, ease of getting started, and ease of use—same as the rest of the project—so setting up a more complex tracking system probably doesn't make a lot of sense here. We can always come back for quantitative questions later.

Does this make sense?
